docs/partials-common/zephyr-install-golioth-firmware-sdk-for-windows.md

```diff
@@ -5,8 +5,7 @@ import TabItem from '@theme/TabItem';
 
     ```shell
     cd c:\
-    west init -m https://github.com/golioth/golioth-zephyr-sdk.git --mf west-zephyr.yml golioth-zephyr-workspace
-    cd golioth-zephyr-workspace
+    west init -m https://github.com/golioth/golioth-firmware-sdk.git --mf west-zephyr.yml golioth-zephyr-workspace
```

Comment:
Should we add `--mr v0.9.0` ?

Reply:
Ah, yes. Good catch! I'll add this to both partials. Thanks.